app/views/instructor_notification_mailer/email.html.haml

```diff
@@ -0,0 +1,26 @@
+%link{rel: 'stylesheet', href:'/mailer.css'}
+%table.row
+  %tbody
+    %tr
+      %th
+        %table
+          %tr
+            %td.main-content
+              %h1.headline
+                = "#{@subject}"
+              %p.paragraph
+                = "Course: "
+                %a{href: @course_link}= @course.title
+              %p.paragraph
+                = "Dear Instructors,"
+              %p.paragraph
+                = "#{@message}"
+
+        %table
+          %tr
+            %td.main-content
+              %p.paragraph
+                Best regards,
```

Comment:
I'd like this to say: "This is a message from Wiki Education staff for your course [course title linking to course_link]."

It should be styled differently as well, probably in a smaller font, to make it more clear which parts of the email are a human-written message and which parts are from a template.

Reply:
Variant 1: Only Text Size Smaller
![image](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/61202986/5ca5d994-242f-4a25-a522-9d5dfccd70db)

Variant 2: Added a `.footer` class in stylesheet changing size, color, font.
![image](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/61202986/310f6259-97bf-4d72-bb6a-5052a4cd961d)

Variant 3: Made main message black and increase `font-weight` to normal.  
![image](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/61202986/24a2a10f-067b-4263-8d13-dbc019a5a1fe)

let me know which one of these satisfies the theme @ragesoss

Reply:
I think the .footer class one looks the best. I suggest using that one, plus adding additional spacing above and below the main message.

Also, there's no need for the 'Best regard, The Wiki Education team'.

The admin can optionally include a personal closing line, but I would rather not hard-code one.

Reply:
Done!
![image](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/61202986/e250acc3-0aa6-4a60-9abe-abfc2bcccc3c)

Reply:
Perfect, this looks great.